gulp-tasks/project.js

```diff
@@ -59,7 +61,9 @@ function merge(source, dependencies) {
     const mergedFiles = mergeStream(source(), dependencies());
 
     return new Promise(resolve => {
-      mergedFiles.pipe(gulp.dest(buildPath))
+      mergedFiles
+        .pipe(gulpif(/\.html$/, gulpreplace('http://localhost:9000/api', 'https://ch.tudelft.nl/payments/api')))
```

Comment:
I think this is a great and clean solution for this problem, I can't come up with a better way :+1: